doc/contrib.rst

```diff
@@ -127,3 +125,49 @@ HTML output by typing, in a shell:
     make html
     firefox _build/html/index.html
 
+
+.. _test_config:
+
+Testing a benchmark
+-------------------
+
+To ensure that the benchmark are reproducible and can be run by other, benchopt provides a set of tools to test that the benchmark is properly formatted, and that it can be installed and run easily.
```

Comment:
```suggestion
To ensure that the benchmark is reproducible and can be run by others, benchopt provides a set of tools to test that the benchmark is properly formatted and that it can be installed and run easily.
```